doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -322,39 +322,22 @@ documentation similarly as with test libraries.
-   :class: example
-
-   +---------------+-----------------------------------------+-----------------------------------+
-   |    Setting    |                  Value                  |               Value               |
-   +===============+=========================================+===================================+
-   | Documentation | Resource file for demo purposes.        |                                   |
-   +---------------+-----------------------------------------+-----------------------------------+
-   | ...           | This resource is only used in an example| and it doesn't do anything useful.|
-   +---------------+-----------------------------------------+-----------------------------------+
-
-.. table::
-   :class: example
-
-   +--------------+------------------+------------------------+-------------------------------+
-   |    Keyword   |      Action      |         Argument       |            Argument           |
-   +==============+==================+========================+===============================+
-   | My Keyword   | [Documentation]  | Does nothing           |                               |
-   +--------------+------------------+------------------------+-------------------------------+
-   |              | No Operation     |                        |                               |
-   +--------------+------------------+------------------------+-------------------------------+
-   |              |                  |                        |                               |
-   +--------------+------------------+------------------------+-------------------------------+
-   | Your Keyword | [Arguments]      | ${arg}                 |                               |
-   +--------------+------------------+------------------------+-------------------------------+
-   |              | [Documentation]  | Takes one argument and | | Example:\\n                 |
-   |              |                  | \*does nothing\* with  | | \| Your Keyword \| xxx \|\\n|
-   |              |                  | it.\\n                 | | \| Your Keyword \| yyy \|\\n|
-   +--------------+------------------+------------------------+-------------------------------+
-   |              | No Operation     |                        |                               |
-   +--------------+------------------+------------------------+-------------------------------+
+Possible variables in resource files can not be documented.
+
+.. sourcecode:: robotframework
+
+   *** Settings ***
+   Documentation    Resource file for demo purposes.
+   ...              This resource is only used in an example and it doesn't do anything useful
+
+   *** Keywords ***
+   My Keyword
+       [Documentation]   Does nothing
+       No Operation
+   Your Keyword
+       [Arguments]  ${arg}
+       [Documentation]   Takes one argument and *does nothing* with it.\n Examples:\n Your Keyword   xxx\n Your Keyword   yyy  
+       No Operation
```

Comment:
- Empty row between tests.
- Doc should be split to rows instead of using `\n`.
- Need to think how to format example.